src/services/ReminderEmailProcessor.ts

```diff
@@ -68,10 +70,16 @@ export class ReminderEmailProcessor {
   		}
 
   		const sendEmailPromises = usersToRemind
-  			.filter((user): user is { sessionId: any; emailAddress: string } => user !== null)
-  			.map(async ({ sessionId, emailAddress }) => {
+  			.filter((user): user is { sessionId: any; emailAddress: string; firstName: string; lastName: string; documentUsed: string } => user !== null)
+  			.map(async ({ sessionId, emailAddress, firstName, lastName, documentUsed }) => {
   				try {
-  					await this.f2fService.sendToGovNotify(buildReminderEmailEventFields(emailAddress));
+  					if (firstName && lastName && documentUsed) {
+  						this.logger.info("Sending Dynamic Reminder Email", { sessionId });
+  						await this.f2fService.sendToGovNotify(buildDynamicReminderEmailEventFields(emailAddress, firstName, lastName, documentUsed));
+  					} else { 
+  						this.logger.info("Sending Static Reminder Email", { sessionId });
```

Comment:
Do we want a check for `emailAddress` for this case or is that covered somewhere else?

Reply:
All fields are checked [here](https://github.com/alphagov/di-ipv-cri-f2f-api/blob/9d486de64066538111754b5974a76db43e1a4e35/src/services/SendEmailProcessor.ts#L33) this is just checking if there's a fName, sNmae and documentType then send the dynamic one but email should always be there at this point